Validation/RecoParticleFlow/plugins/PFJetDQMPostProcessor.cc

```diff
@@ -83,43 +86,87 @@ void PFJetDQMPostProcessor::dqmEndJob(DQMStore::IBooker& ibook_, DQMStore::IGett
     iget_.setCurrentFolder(genjetDir);
     std::vector<std::string> sME_genjets = iget_.getMEs();
     std::for_each(sME_genjets.begin(), sME_genjets.end(), [&](auto& s) { s.insert(0, genjetDir); });
-    //for (unsigned int i=0; i<sME_genjets.size(); i++) std::cout << sME_genjets[i] << std::endl;
+
+    iget_.setCurrentFolder(offsetDir);
+    std::vector<std::string> sME_offset = iget_.getMEs();
+    std::for_each(sME_offset.begin(), sME_offset.end(), [&](auto& s) { s.insert(0, offsetDir); });
 
     iget_.setCurrentFolder(jetResponseDir[idir]);
     std::vector<std::string> sME_response = iget_.getMEs();
     std::for_each(sME_response.begin(), sME_response.end(), [&](auto& s) { s.insert(0, jetResponseDir[idir]); });
-    //for (unsigned int i=0; i<sME_response.size(); i++) std::cout << sME_response[i] << std::endl;
 
     iget_.setCurrentFolder(jetResponseDir[idir]);
 
     double ptBinsArray[ptBins.size()];
     unsigned int nPtBins = ptBins.size() - 1;
     std::copy(ptBins.begin(), ptBins.end(), ptBinsArray);
-    //for(unsigned int ipt = 0; ipt < ptBins.size(); ++ipt) std::cout << ptBins[ipt] << std::endl;
 
     std::string stitle;
     char ctitle[50];
+    std::vector<MonitorElement*> vME_efficiency;
+    std::vector<MonitorElement*> vME_purity;
     std::vector<MonitorElement*> vME_presponse;
+    std::vector<MonitorElement*> vME_presponse_mean;
+    std::vector<MonitorElement*> vME_presponse_median;
     std::vector<MonitorElement*> vME_preso;
     std::vector<MonitorElement*> vME_preso_rms;
+    std::vector<MonitorElement*> vME_ratePUJet;
 
     MonitorElement* me;
     TH1F* h_resp;
-    TH1F* h_genjet_pt;
+    TH1F *h_genjet_pt, *h_genjet_matched_pt;  // *h_genjet_unmatched_pt;
+    TH1F *h_recojet_pt, *h_recojet_matched_pt, *h_recojet_unmatched_pt;
+
+    stitle = offsetDir + "mu";
+    std::vector<std::string>::const_iterator it = std::find(sME_offset.begin(), sME_offset.end(), stitle);
+    if (it == sME_offset.end())
+      std::cout << "ERROR: Offset Folder does not exist." << std::endl;
```

Comment:
Sorry, couts are discouraged

Reply:
I have substituted the comment with `continue;`.